code/modules/mining/machine_vending.dm

```diff
@@ -35,9 +35,6 @@
 		/obj/item/wormhole_jaunter = 3,
 		/obj/item/kinetic_crusher = 1,
 		/obj/item/gun/energy/kinetic_accelerator = 3,
-		/obj/item/pinpointer/deepcore = 5,		// WS edit - Deepcore
-		/obj/item/pinpointer/deepcore/advanced = 2,		// WS edit - Deepcore
-		/obj/item/deepcorecapsule = 3,		// WS edit - Deepcore
```

Comment:
These never got replaced with normal mining scanners, was this intentional?

Reply:
gif of the anime girl going beh

no it wasn't